cypress/component/ComponentName.cy.js

```diff
@@ -0,0 +1,5 @@
+describe('ComponentName.cy.js', () => {
```

Comment:
Can this be removed? Seems auto-generated/doesn't provide value

Reply:
Removed.